woocommerce/wc-admin
# Initial paypal plugin replacement

Fixes #5788 

Heads up, this will likely get replaced by this PR soon: https://github.com/woocommerce/woocommerce-admin/pull/6257

This PR replaces the use of the Paypal Express Checkout plugin with the Paypal payments plugin.
This is the very basic version, where it only installs and enables the plugin, it then provides a link to the set up settings for the Paypal plugin.

A more interactive version will be added in another PR.

### Accessibility




### Screenshots

![new-paypal-plugin](https://user-images.githubusercontent.com/2240960/106659430-1acaa800-6575-11eb-9f32-2b7cffa1c54c.gif)

### Detailed test instructions:

- Create new Woo store and finish the onboarding wizard
- Go to the **Set up payments** task and click **Set up** on the Paypal plugin. (It should say Paypal Payments)
- It should automatically start the **Install** step, and install the [Paypal Payments](https://woocommerce.com/products/woocommerce-paypal-payments/) plugin.
- Click on the **store settings** on the next tab, this should direct you to the Paypal settings tab
- Click on **Continue**, this will bring you back to the payment options.

Onboarding
- Follow the **store settings** link in the connect step
- Click **Connect to Paypal**, this should open a popup and allow you to either create a new account, or use your existing.
  - Make sure you select the same country your store is in.
- Finish the connection modal, the last option should close the modal and redirect you back to the settings
- Navigate back to **Woocommerce > Home > Set up payments**
- The **Set up** button should be gone, and instead there is a toggle to enable/disable the plugin
- The enable/disable button should be correctly reflected in the Woocommerce payment settings screen as well.

Quick client test (With paypal payments enabled)
- Make sure you have added a product and store homescreen
- Navigate to one of the products and add it to the cart
- Click **go to checkout**
- Paypal should be one of the options to pay
- Filling in your billing/shipping info then click pay with **Paypal**
- The paypal pay window should pop up correctly without errors.
